src/main/java/org/terasology/customOreGen/PocketStructureDefinition.java

```diff
@@ -133,9 +136,9 @@ public DiffusePocketStructure(Transform transform, Random random, Vector3ic chun
             noiseLevels = (maxSize <= 1) ? 0 : (int) (java.lang.Math.log(maxSize) / java.lang.Math.log(2) + 0.5F);
 
             // store transforms
-            mat = transform.clone();
+            mat = new Matrix4f(transform);
             if (transform.determinant() != 0) {
-                invMat = transform.inverse();  // note - this alters the transform argument
+                invMat = transform.invert(new Matrix4f());  // note - this alters the transform argument
```

Comment:
This does not actually alter the `transform` argument. This is fine, but the comment should be removed.